utils.go

```diff
@@ -75,19 +76,48 @@ func loadHistorySchema(IdType *field.TypeInfo, entqlEnabled bool) (*load.Schema,
 	return historySchema, nil
 }
 
-func getUpdatedByField(updatedByValueType string) (*load.Field, error) {
+func getUpdatedByField(updatedByValueType string, entgqlEnabled bool) (*load.Field, error) {
 	if updatedByValueType == "String" {
-		return load.NewField(field.String("updated_by").Optional().Nillable().Immutable().Descriptor())
+		f := field.String("updated_by").Optional().Nillable().Immutable()
+		if entgqlEnabled {
+			f = f.Annotations(entgql.Annotation{Type: "ID"})
+		}
+		return load.NewField(f.Descriptor())
 	}
 	if updatedByValueType == "Int" {
-		return load.NewField(field.Int("updated_by").Optional().Nillable().Immutable().Descriptor())
+		// return load.NewField(field.Int("updated_by").Optional().Nillable().Immutable().Descriptor())
```

Comment:
Just making sure, this can get deleted now right? There's not condition where it should be used?